applications/ShapeOptimizationApplication/python_scripts/custom_ios/wrl_io.py

```diff
@@ -0,0 +1,63 @@
+# ==============================================================================
+#  KratosShapeOptimizationApplication
+#
+#  License:         BSD License
+#                   license: ShapeOptimizationApplication/license.txt
+#
+#  Main authors:    Geiser Armin, https://github.com/armingeiser
+#
+# ==============================================================================
+
+import KratosMultiphysics as KM
+from .wrl_reader import read_shapes
+
+
+def _rename_to_valid_name(model_part, shape):
+    name = shape.name
+
+    ref_name = name
+    i = 2
+    while model_part.HasSubModelPart(name):
+        name = ref_name +"_{}".format(i)
+        i += 1
+
+    if shape.name != name:
+        KM.Logger.PrintWarning("ShapeOpt", "WrlIO: Name of the sub model part has been changed from '{}' to '{}' in order "\
+            "to avoid name clashes!".format(shape.name, name))
+        shape.name = name
+
+
+class WrlIO:
+
+    def __init__(self, file_name):
+
```

Comment:
It's very minor cosmetics. But you should remove that extra line after the function declaration.  Also in the other functions, you started directly after the declaration.